srcpkgs/engrampa/template

```diff
@@ -9,8 +9,9 @@ makedepends="file-devel gtk+3-devel json-glib-devel libSM-devel libarchive-devel
  libcaja-devel"
 depends="desktop-file-utils"
 short_desc="Archive manipulator for MATE"
-maintainer="Orphaned <orphan@voidlinux.org>"
+maintainer="skmpz <dem.procopiou@gmail.com>"
 license="GPL-2.0-or-later"
 homepage="https://mate-desktop.org"
 distfiles="https://pub.mate-desktop.org/releases/${version%.*}/${pkgname}-${version}.tar.xz"
-checksum=d415f486edc9a5474024b3928ff1127ba4534a703a60993a3912fffcf0988a8d
+checksum=fe24e04efb34a67374d2a46a3bf2a0a445b77e0ec43216e4c3f14e5cd0b7722a
+nocross="cannot run test program while cross compiling"
```

Comment:
this is unacceptable, please file an issue upstream. commit that broke it: https://github.com/mate-desktop/engrampa/commit/25b2bdbcfb95c91a33fd62370cac28a863567c3f

Reply:
If they want to keep their (imo terrible) test, they should at least run it with host CC.

Reply:
Alternatively, they can just check for the two possible ZSTD values and not require all this machinery.